sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.2.1 (2022-08-12)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Updated `azure-communication-common` version
```

Comment:
```suggestion
### Other Changes
- Updated `azure-communication-common` version.
```

Reply:
None of the other releases did it this way, take a look at:

## 1.0.3 (2021-06-28)
## 1.0.2 (2021-06-09)

Don't mind doing it but will be inconsistent with the other releases in the change log.